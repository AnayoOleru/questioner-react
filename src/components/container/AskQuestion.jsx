import React, { Component, Fragment } from 'react';
import Header from '../presentation/Header';
import Footer from '../presentation/Footer';

class AskQuestion extends Component {
  render() {
    return (
      <Fragment>
        <Header />
        <div className='AskQuestion'>
          <div className='ask-question-area'>
            <div className='container'>
              <div className='meetup-title-area'>
                <div className='meetup-title'>
                    NIGERIA MODERN EXCEL & POWER BI USER GROUP
                </div>
              </div>
              <div className='ask-question-form-area'>
                <h3 className='ask-question-form-text'>Ask A Question</h3>
                <form>
                  <div className='ask-question-input-section'>
                    <input
                      type='text'
                      placeholder='Title'
                      required
                    />
                  </div>
                  <div className='ask-question-input-section'>
                    <textarea required placeholder='Body' l />
                  </div>
                  <div className='ask-question-form-submit'>
                    <button type='submit'>Submit</button>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
        <Footer />
      </Fragment>
    );
  }
}

export default AskQuestion;
