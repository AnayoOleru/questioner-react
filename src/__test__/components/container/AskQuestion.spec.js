import React from 'react';
import { shallow } from 'enzyme';
import AskQuestion from '../../../components/container/AskQuestion';

describe('Ask Question Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<AskQuestion />);
  });
  test('Ask question component should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
