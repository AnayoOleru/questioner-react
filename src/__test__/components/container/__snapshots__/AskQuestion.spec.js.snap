// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ask Question Component Ask question component should render 1`] = `ShallowWrapper {}`;
